### DISCUSSION TITLE
Issue template on github

### DISCUSSION THREAD

**Discussion by Faq** [nzbgetcom/nzbget#157] [Ideas]
Hello,
Checked some issues, saw that you don't use issue template.
To avoid asking same questions, like: what platform and version used, or other info. 

Suggest to implement it similar how it was done in this project: https://github.com/TrinityCore/TrinityCore/issues
Just click "New Issue" button and enjoy.

ref. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

**Comment by luckedea** [comment]
@Faq Yes thank you for the tip, I'll set it up soon - previously I was looking forward to collect as much raw feedback as possible since it defines development priorities, and it would tell what kind of assistance we can give to users asking for help.

**Comment by luckedea** [comment]
Added issues templates. Not ideal, but good start. I'll be adjusting them based on issues we receive.
Thank you @Faq
